<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TapeCopyClass</class>
 <widget class="QDialog" name="TapeCopyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HetMap</string>
  </property>
  <widget class="QGroupBox" name="FileBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>501</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Tape</string>
   </property>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseFileButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ScsiBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Tape</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/st</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="fileToScsiRadio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy From File to SCSI</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="scsiToFileRadio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy From SCSI to File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
